Makefile

```diff
@@ -1,6 +1,9 @@
+include variables.mk
+
 # Image URL to use all building/pushing image targets
 # TODO refactor after csi-baremetal-operator renaming https://github.com/dell/csi-baremetal/issues/341
```

Comment:
Please, remove this TODO and close the issue

Reply:
Fixed.